### PR TITLE
Afterpay countries

### PR DESCRIPTION
**Pull request by LeasoftKonstantinShapoval**

### CODE REVIEW

File: upload/admin/language/de-de/extension/payment/emspay_ideal.php [hunk 1]
```diff
@@ -52,4 +52,4 @@
 $_['info_help_klarna_ip_filter'] = 'Wenn sie eingegeben werden, wird nur für diese IPs die Zahlungsmethode angezeigt. (Beispiel: 128.0.0.1, 255.255.255.255)';
 $_['entry_klarna_ip_filter'] = 'IP-Filterung:';
 $_['entry_klarna_test_api_key'] = 'Test API schlüssel:';
-$_['info_help_klarna_test_api_key'] = 'Wenn eingegeben, wird dieser API-Schlüssel nur zu Testzwecken verwendet werden.';
\ No newline at end of file
+$_['info_help_klarna_test_api_key'] = 'Wenn eingegeben, wird dieser API-Schlüssel nur zu Testzwecken verwendet werden.';
```
Comment (tiutiunnykleasoft) [inline upload/admin/language/de-de/extension/payment/emspay_ideal.php]: Remove empty string, that file isn't changed

File: upload/admin/language/fr-fr/extension/payment/emspay_ideal.php [hunk 2]
```diff
@@ -52,4 +52,4 @@
 $_['info_help_klarna_ip_filter'] = 'Si cette champs est rempli, uniquement pour ces adresses IP le mode de paiement sera affiché. (Par Ex: 128.0.0.1, 255.255.255.255)';
 $_['entry_klarna_ip_filter'] = 'le filtrage IP:';
 $_['entry_klarna_test_api_key'] = 'clé API de Test:';
-$_['info_help_klarna_test_api_key'] = 'Si entré, cette clé API sera utilisée uniquement à des test.';
\ No newline at end of file
+$_['info_help_klarna_test_api_key'] = 'Si entré, cette clé API sera utilisée uniquement à des test.';
```
Comment (tiutiunnykleasoft) [inline upload/admin/language/fr-fr/extension/payment/emspay_ideal.php]: Remove empty string

File: upload/admin/language/nl-nl/extension/payment/emspay_ideal.php [hunk 3]
```diff
@@ -52,4 +52,4 @@
 $_['info_help_klarna_ip_filter'] = 'U  kunt specifieke IP adressen opgeven waarvoor de betaalmethode zichtbaar is, bv als u wilt testen  (bijvoorbeeld: 128.0.0.1, 255.255.255.255). Vult u niets in dan is de betaalmethode voor alle IP adressen zichtbaar.';
 $_['entry_klarna_ip_filter'] = 'IP-Filteren:';
 $_['entry_klarna_test_api_key'] = 'Test API Key:';
-$_['info_help_klarna_test_api_key'] = 'Vul hier de API Key in van uw test webshop.';
\ No newline at end of file
+$_['info_help_klarna_test_api_key'] = 'Vul hier de API Key in van uw test webshop.';
```
Comment (tiutiunnykleasoft) [inline upload/admin/language/nl-nl/extension/payment/emspay_ideal.php]: remove empty string

File: upload/catalog/model/extension/payment/emspay_afterpay.php [hunk 4]
```diff
@@ -2,17 +2,17 @@
...
         $query = $this->db->query("SELECT * 
-            FROM ".DB_PREFIX."zone_to_geo_zone 
-            WHERE geo_zone_id = '".(int) $this->config->get('emspay_afterpay_geo_zone_id')."' 
-            AND country_id = '".(int) $address['country_id']."' 
-            AND (zone_id = '".(int) $address['zone_id']."' 
+            FROM " . DB_PREFIX . "zone_to_geo_zone 
+            WHERE geo_zone_id = '" . (int)$this->config->get('emspay_afterpay_geo_zone_id') . "' 
+            AND country_id = '" . (int)$address['country_id'] . "' 
+            AND (zone_id = '" . (int)$address['zone_id'] . "' 
             OR zone_id = '0');"
```
Comment (tiutiunnykleasoft) [inline upload/catalog/model/extension/payment/emspay_afterpay.php]: Do these changes actually change something?
If not, remove them

File: upload/system/library/emshelper.php [hunk 5]
```diff
@@ -658,6 +658,22 @@ public static function ipIsEnabled($ipList)
         return true;
     }
 
+    /**
+     * @param $countryList
+     * @return bool
+     */
+    public static function CountryValidator($countryList, $billingAddress)
+    {
+        if ($countryList !== '' ) {
```
Comment (tiutiunnykleasoft) [inline upload/system/library/emshelper.php]: https://github.com/emspay/ems-online-opencart-2/pull/7#discussion_r465632188

File: upload/system/library/emshelper.php [hunk 6]
```diff
@@ -658,6 +658,22 @@ public static function ipIsEnabled($ipList)
         return true;
     }
 
+    /**
+     * @param $countryList
+     * @return bool
+     */
+    public static function CountryValidator($countryList, $billingAddress)
```
Comment (tiutiunnykleasoft) [inline upload/system/library/emshelper.php]: https://github.com/emspay/ems-online-opencart-2/pull/7#discussion_r465632814

File: upload/admin/view/template/extension/payment/emspay.twig [hunk 7]
```diff
@@ -284,6 +284,24 @@
...
+                            </label>
+                            <div class="col-sm-10">
+                                <input type="text" name="payment_{{ emspay_module }}_country_access" id="input-country-access"
+                                       value="{{ _context['payment_'~emspay_module~'_country_access'] }}"
+                                       placeholder="{{ info_example_country_access }}"
+                                       class="form-control"/>
+                            </div>
+                        </div>
+                    {% endif %}
+
```
Comment (tiutiunnykleasoft) [inline upload/admin/view/template/extension/payment/emspay.twig]: https://github.com/emspay/ems-online-opencart-2/pull/7#discussion_r465779885

**Review by tiutiunnykleasoft — CHANGES_REQUESTED** [review]

File: upload/catalog/model/extension/payment/emspay_afterpay.php (hunk 4 above)
Comment (LeasoftKonstantinShapoval) [reply upload/catalog/model/extension/payment/emspay_afterpay.php]: Roll backed.

**Review by tiutiunnykleasoft — APPROVED** [review]

**Pull request merged**